app/gateway-manager/data-plane-reference.md

```diff
@@ -269,7 +266,81 @@ rows:
+The default directory for {{site.base_gateway}} logs is [`/usr/local/kong/logs`](/gateway/configuration/#log-level).
+
+    If you find any of the following errors:
+
+    * Data Plane node failed to connect to the Control Plane.
+    * Data Plane node failed to ping the Control Plane.
+    * Data Plane node failed to receive a ping response from the Control Plane.
+
+    You may have an issue on the host network where the node resides.
+    Diagnose and resolve the issue, then restart the node and check the sync status in the Gateway Manager.
+
+If the logs show a license issue, or if you are unable to resolve sync issues using the above methods, contact [Kong Support](https://support.konghq.com/).
+
+### Missing functionality
+
+**Problem:** If a {{site.konnect_short_name}} feature isn’t working or isn't available on your Data Plane node, the version may be out of date.
+
+**Solution:** Check that your Data Plane nodes are up to date, and update them if they are not. 
+For Dedicated Cloud Gateways, see the [upgrade documentation](#upgrade-data-planes).
+
+If running {{site.base_gateway}} in hybrid mode, check that Data Plane node versions are up to date:
+
+1. Open [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), then open your Control Plane.
+
+1. Select **Data Plane Nodes** from the side menu, then click the **New Data Plane Node** button.
+
+1. Check the {{site.base_gateway}} version in the code block. 
+This is the version that the {{site.konnect_short_name}} Control Plane is running.
+
+1. Return to the Data Plane nodes page.
+
+1. Check the Data Plane node versions in the table. 
+If you see a node running an older version of {{site.base_gateway}}, your Data Plane node may need [upgrading](#upgrade-data-planes).
+
+If your version is up to date but the feature still isn't working, contact [Kong Support](https://support.konghq.com/).
+
+### Kubernetes Data Plane node installation does not work
+
+**Problem:** You followed the Kubernetes installation instructions in Gateway Manager but your Data Plane node isn't connecting.
+ 
+**Solution:** Check your deployment logs for errors:
+
+```bash
+kubectl logs deployment/my-kong-kong -n kong
+```
+
+If you find any errors and need to update `values.yaml`, make your changes, save the file, then reapply the configuration by running the Helm `upgrade` command:
+
+```bash
+helm upgrade my-kong kong/kong -n kong \
+  --values ./values.yaml
+```
+
+### Connect to a Data Plane located behind a non-transparent forward proxy
+
+**Problem:** You need to connect the Control Plane to a Data Plane node that is behind a non-transparent forward proxy.
+
+**Solution:** Configure each {{site.base_gateway}} Data Plane node to authenticate with the proxy server and allow traffic through.
+For more information, learn how to [use a forward proxy to secure communication across a firewall](/gateway/cp-dp-communication/#use-a-forward-proxy-to-secure-communication-across-a-firewall).
```

Comment:
```suggestion
```
This one feels more like a different "feature" rather than troubleshooting, so I recommend moving it to maybe an h3 in the "Forward proxy section"?